core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,38 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
```

Comment:
Yeah I say let's remove it @whmii, that way we can find the need for a feature later to add what feels natural.